publications/views.py

```diff
@@ -212,13 +196,19 @@ def data(request):
     if not os.path.exists(json_path) or not os.path.exists(gpkg_path):
         regenerate_geopackage_cache()
 
-    geojson_size = os.path.getsize(json_path)
-    geopackage_size = os.path.getsize(gpkg_path)
+    if os.path.exists(json_path):
+        geojson_size = humanize.naturalsize(os.path.getsize(json_path), binary=True)
+    else:
+        geojson_size = None
+
+    if os.path.exists(gpkg_path):
+        geopackage_size = humanize.naturalsize(os.path.getsize(gpkg_path), binary=True)
+    else:
+        geopackage_size = None
     ts = os.path.getmtime(json_path)
```

Comment:
You only take the time of one format here, which is fine because all files _should_ be created at he same time. However, what if they are not?

Please load the times of all files here and compare them that they are about the same time, within an hour is fine. Add a log statement here with level `warning` if the difference is more than an hour. 
The code can then proceed as it is, but at least we have a hint in the logs that something may be wrong.